src/utils/passwordService.js

```
@@ -0,0 +1,29 @@
+      const allChars = uppercase + lowercase + numbers + special;
+      for (let i = password.length; i < 12; i++) {
+          password += allChars[Math.floor(Math.random() * allChars.length)];
+      }
+
+      // Shuffle the password to make it more random
+      return password
+        .split('')
+        .sort(() => Math.random() - 0.5)
+        .join('');
+  };
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)